.github/workflows/main.yml

```diff
@@ -100,17 +102,19 @@ jobs:
           ARIESVCX_SOURCE_HASH=${{ hashFiles('aries_vcx') }}
           AGENCY_CLIENT_HASH=${{ hashFiles('agency_client') }}
 
-          LIBVCX_DOCKERFILE_HASH=${{ hashFiles('ci/libvcx.dockerfile') }}
           ALPINE_CORE_DOCKERFILE_HASH=${{ hashFiles('ci/alpine_core.dockerfile')}}
+          LIBVCX_DOCKERFILE_HASH=${{ hashFiles('ci/libvcx.dockerfile') }}
+          LIBVCX_TESTER_DOCKERFILE_HASH=${{ hashFiles('ci/libvcx-tester.dockerfile') }}
           CODECOV_DOCKERFILE_HASH=${{ hashFiles('ci/libvcx-codecov.dockerfile') }}
 
           JAVA_WRAPPERS_HASH=${{ hashFiles('wrappers/java') }}
           LIBVCX_NODE_WRAPPERS_HASH=${{ hashFiles('wrappers/node') }}
           LIBVCX_NODE_AGENT_HASH=${{ hashFiles('agents/node') }}
 
           POOL_DOCKERFILE_HASH=${{ hashFiles('ci/indy-pool.dockerfile')}}
-
-          LIBVCX_HASH=${LIBVCX_SOURCE_HASH:0:11}-${ARIESVCX_SOURCE_HASH:0:11}-${LIBVCX_NODE_WRAPPERS_HASH:0:11}-${LIBVCX_NODE_AGENT_HASH:0:11}-${LIBVCX_DOCKERFILE_HASH:0:11}-${AGENCY_CLIENT_HASH:0:11}
+          
+          LIBVCX_HASH=${LIBVCX_DOCKERFILE_HASH:0:11}-${LIBVCX_SOURCE_HASH:0:11}-${ARIESVCX_SOURCE_HASH:0:11}-${AGENCY_CLIENT_HASH:0:11}
+          LIBVCX_TESTER_HASH=${LIBVCX_TESTER_DOCKERFILE_HASH}-${LIBVCX_HASH}-${LIBVCX_NODE_WRAPPERS_HASH:0:11}-${LIBVCX_NODE_AGENT_HASH:0:11}
```

Comment:
Please explicitly slice all the hashes for the length to be bounded by an explicit number.